### PR TITLE
Add draft README text

### PR DESCRIPTION
**Pull request by davschne**

### CODE REVIEW

**Review by kbeame — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,36 @@
 # Choroscope
+
+## What is Choroscope?
+
+Choroscope is a visualization platform for geospatial data. Created to showcase results for the [Local Burden of Disease (LBD)](http://www.healthdata.org/lbd) project at the [Institute for Health Metrics and Evaluation (IHME)](http://www.healthdata.org), Choroscope provides a flexible format for exploring and sharing data linked to geography.
+
+## How does Choroscope work?
+
+The platform ingests a geospatial data set with an accompanying configuration file specifying the shape of the data and details of how it should be displayed. A database creation tool, `choroscope-db`, builds a geospatial database from the data, and an application server, `choroscope-api`, makes it available as a web API. A frontend web application, `choroscope-app`, uses this API to display both choropleths and highly detailed image-like maps - for which pixel colorization is done in the browser with a custom WebGL renderer. The configuration language, defined in `choroscope-spec`, allows the addition of other visualizations, like line and bar graphs, to supplement the primary map-based view.
```
Comment (Ryshackleton) [inline README.md]: A simple architectural diagram with a few boxes showing the db, app, and spec would go a long way towards showing the different pieces and their relationships. I'm thinking you might have arrows and things that are labeled "reads", "builds", "configures". It would help me!

File: README.md [hunk 2]
```diff
@@ -1 +1,36 @@
...
+
+## How did Choroscope come to be?
+
+Choroscope grew out of the Local Burden of Disease (LBD) project at IHME, and its features and design strongly reflect the needs of that project. Above all, LBD needed a flexible and open-ended tool to visualize a large number of data sets of differing shapes and with differing needs. As of this writing, the LBD application is being used to visualize population health estimates accompanying eight scientific publications, with many more in the works. Because of the tool's flexibility, we the developers of the software came to realize that the underlying platform could, in principle, be used to visualize _any_ compatible data set - it wasn't limited to LBD estimates or even health data. In the spirit of open sharing of information embraced by LBD, IHME, and the scientific community generally, we wanted to offer the core platform as open-source software so that others could use it to visualize their geospatial data sets, ultimately helping us all to better understand our world.
+
+## What does the name mean?
```
Comment (Ryshackleton) [inline README.md]: I would put this section right below "What is Choroscope"?

File: README.md [hunk 3]
```diff
@@ -1 +1,36 @@
...
+- `choroscope-app` is a frontend web application that visualizes the data provided by `choroscope-api`. Written in [TypeScript](http://www.typescriptlang.org/), the application leverages the mapping framework [Leaflet](https://leafletjs.com/) as well as a variety of other open-source technologies. It also provides a custom [WebGL-based renderer](https://github.com/ihmeuw/leaflet.tilelayer.glcolorscale) for visualizing pixel data.
+
+For the Local Burden of Disease project, we've created additional resources for deploying the Choroscope application and its databases as containerized microservices. Our intent is that the platform itself be agnostic as to how and where it's deployed, but to simplify the setup process for people who want to create their own instances of Choroscope, we also plan to publish generalized deployment resources, probably in the form of a [Helm Chart](https://helm.sh/) for deployment in [Kubernetes](https://kubernetes.io/).
```
Comment (Ryshackleton) [inline README.md]: This sentence runs on a bit: "Our intent is that the platform itself be agnostic as to how and where it's deployed, but to simplify the setup process for people who want to create their own instances of Choroscope, we also plan to publish generalized deployment resources, probably in the form of a Helm Chart for deployment in Kubernetes."

Maybe reword as follows:
"Our intent is that the platform itself be agnostic as to how and where it's deployed. In the future we plan to simplify the Choroscope setup process by publishing generalized deployment resources to assist with instantiation of the services, probably in the form of a Helm Chart for deployment in Kubernetes."

File: README.md [hunk 4]
```diff
@@ -1 +1,36 @@
...
+
+## Where can I see Choroscope in action?
+
+You can see a working instance of Choroscope in IHME's [Local Burden of Disease application](https://vizhub.healthdata.org/lbd). This application uses Choroscope to visualize estimates of population health. The Choroscope platform, though, is by no means limited to showing this kind of data. It can be used to visualize any data set, with any dimensions, using the same format. The IHME branding and navigation you see in the LBD application are in a layer separate from the core platform; they won't be included in the open-source offering.
```
Comment (Ryshackleton) [inline README.md]: I don't like the "..., though, ...".  Maybe change:
"This application uses Choroscope to visualize estimates of population health. The Choroscope platform, though, is by no means limited to showing this kind of data."

...to: "While the Local Burden of Disease application uses Choroscope to visualize estimates of population health, the Choroscope platform is by no means limited to showing this kind of data."

**Review by Ryshackleton — APPROVED** [review]
Looks good! I provided a few rewording suggestions, and I recommend creating an architecture diagram, even if it is very very simple.

File: README.md (hunk 2 above)
Comment (davschne) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (davschne) [reply README.md]: Done, though I changed the wording a bit.

File: README.md (hunk 4 above)
Comment (davschne) [reply README.md]: Done.

File: README.md (hunk 1 above)
Comment (davschne) [reply README.md]: Good idea! I'll think about this and see what I can come up with.

Comment (davschne) [reply README.md]: Okay, diagram added.
